web/styles/message_row.css

```diff
@@ -180,6 +184,7 @@ $time_column_min_width: 42px; /* + padding */
         .message_time {
             justify-self: end;
             padding-right: 5px;
+            width: var(--formatted-time-width);
```

Comment:
This is also incorrect; this should be an adjustment to the grid definition and the width of the time column, not a freestanding `width` value on the time element itself.